map/map_tests/kmz_unarchive_test.cpp

```diff
@@ -39,3 +40,13 @@ UNIT_TEST(KMZ_UnzipTest)
     TEST_EQUAL(bm.GetScale(), 0, ("KML wrong scale!"));
   }
 }
+
+UNIT_TEST(Multi_KML_KMZ_UnzipTest)
+{
+  std::string const kmzFile = GetPlatform().TestsDataPathForFile("kml_test_data/BACRNKMZ.kmz");
+  auto const filePaths = GetKMLOrGPXFilesPathsToLoad(kmzFile);
+  // Check that doc.kml (could be renamed to docN.kml) is skipped
```

Comment:
Can the root file have _any_ name, not just a doc(N).kml?

Reply:
I haven't seen files/examples with index name other than doc.kml, also looks like a convention: https://stackoverflow.com/a/12897689/21894392 I think we can assume it until we see that some software doesn't follow it.

Reply:
Let's follow [the spec](https://developers.google.com/kml/documentation/kmzarchives):

> Note: For clarity, this page refers to the main KML file within a KMZ archive as doc.kml. This main KML file can have any name, as long as it ends in .kml, and as long as there is only one .kml file.

Reply:
Yes, that is how it works now - doc.kml is ignored only if there is more than one kml file. (as in spec "and as long as there is only one .kml file."). Single file file with any name is accepted.